### DISCUSSION TITLE
How to actually merge it with steam so it will work (achievments etc.)

### DISCUSSION THREAD

**Discussion by PanMenel** [MadDeCoDeR/Classic-RBDOOM-3-BFG#150] [Q&A] [answered]
I tried many times to merge it and i don't know how to make it work, on moddb i saw q&a "Can I still get Steam Achievements with this?

A: Yes, if you download [Open Platform]"
I tried using it but I don't know how it works and now I am asking for help because i have no idea what I'm doing.

**Comment by MadDeCoDeR** [comment, marked as answer]
Hi, how do you try to install it?

For starters to install it you must get the right files from the zip file you have downloaded.
Using on BFA the launch argument +set com_allowConsole 1, after the game is launched press the ~ key to open the console and from there check the lower right after the name of the port what it has.
It should have one of the 3 values:
- win-x64: 64-bit Windows
- win-x86: 32-bit Windows
- linux-x86_64: 64-bit Linux
Depending on what it say get the files from the right folder in the OpenPlatform zip file.
After that put the OpenPlatform dll or .so file to the <game's installation folder>/base
and the steam_api just to the game's installation folder.
Lastly, make a .txt file in the game's installation folder named steam_appid and put the publicly known app Id of the game (208200)

**Reply by PanMenel** [reply]
The only thing I was missing was an open platform dll in the base folder. Thanks for the fast response, Keep up the good work.
And I have one last question: is this mod supporting all achievements, and am I good playing through all games 100%?

**Reply by MadDeCoDeR** [reply]
> The only thing I was missing was an open platform dll in the base folder. Thanks for the fast response, Keep up the good work.
> And I have one last question: is this mod supporting all achievements, and am I good playing through all games 100%?

Technically it supports all of them. But the only issue is how to play in multi-player to get the achievements

**Reply by PanMenel** [reply]
ok, thanks again.
